reco_utils/recommender/deeprec/models/base_model.py

```diff
@@ -46,24 +50,33 @@ def __init__(self, hparams, iterator_creator, graph=None):
 
         # set GPU use with demand growth
         gpu_options = tf.GPUOptions(allow_growth=True)
-        self.sess = tf.Session(graph=self.graph, config=tf.ConfigProto(gpu_options=gpu_options))
+        self.sess = tf.Session(
+            graph=self.graph, config=tf.ConfigProto(gpu_options=gpu_options)
+        )
         self.sess.run(self.init_op)
 
+    @abc.abstractmethod
+    def _build_graph(self):
+        """Subclass will implement this."""
+        pass
+
     def _get_loss(self):
-        """
-        Make loss function, consists of data loss and regularization loss
-        :return:  loss value
+        """Make loss function, consists of data loss and regularization loss
+        
+        Returns:
+            obj: Loss value
         """
         self.data_loss = self._compute_data_loss()
         self.regular_loss = self._compute_regular_loss()
         self.loss = tf.add(self.data_loss, self.regular_loss)
         return self.loss
 
     def _get_pred(self, logit, task):
-        """
-        Make final output as prediction score, according to different tasks.
-        :param logit: base prediction value
-        :param task:  regression or classification
+        """Make final output as prediction score, according to different tasks.
+        
+        Args:
+            logit (obj): Base prediction value.
+            task (str): A task (values: regression/classification)
         :return: transformed score
```

Comment:
Here, you missed one line of reformatting. `Returns:`

Reply:
good catch